### PR TITLE
chore: remove unnecessary line

### PR DESCRIPTION
**Pull request by jrcarlos2000**
## Related Issue

remove unnecessary line, pass tests, no functions from `Position` library are used.

### CODE REVIEW

File: src/PoolManager.sol [hunk 1]
```diff
@@ -80,7 +80,6 @@ contract PoolManager is IPoolManager, ProtocolFees, NoDelegateCall, ERC6909Claim
     using SafeCast for *;
     using Pool for *;
     using Hooks for IHooks;
-    using Position for mapping(bytes32 => Position.State);
```
Comment (hensha256) [inline src/PoolManager.sol:83]: i think youre right. Can you also remove it in the ProxyPoolManager.sol

Comment (0xquantum3labs) [reply src/PoolManager.sol:83]: done ✅

Comment (jrcarlos2000) [reply src/PoolManager.sol:83]: done ✅

Comment (jrcarlos2000) [reply src/PoolManager.sol:83]: @hensha256

**Comment by hensha256** [comment]
Sorry for the delay here! We've now added a snapshot to check bytecode hash in #924, so if CI passes here we know that you dont alter the bytecode and we can merge :)

**Comment by jrcarlos2000** [comment]
thanks @hensha256

**Comment by jrcarlos2000** [comment]
@hensha256 should be good now :)

**Pull request merged**